content/broker/what-do-you-get/_index.en.md

```
@@ -11,26 +11,45 @@ weight: 20
 Altinn Broker offers end-to-end file transfers, from creation, sending, receiving to tracking file transfers. This ensures that all parts of the messaging flow are handled efficiently and securely.
 
 ### Support for large payloads
-A unique feature of Altinn Broker is its ability to handle large data volumes (payloads). We aim to support very large payloads in the near future. This is the solution for those who need to transfer large amounts of information efficiently and securely.
+A unique feature of Altinn Broker is its ability to handle large data volumes (payloads). The system supports file transfers of up to 2 GB with virus scanning, and up to 1600 GB without. This is the ideal solution for those who need to transfer large amounts of information efficiently and securely.
 
-### GUI and API access
-Users have access to all functionalities through both a user-friendly interface (GUI) and via APIs. This provides you with the flexibility to integrate Altinn Broker into your own systems, allowing you to automate your processes.
+### Security
+Use of security mechanisms to protect content, you will find more information in this article: https://learn.microsoft.com/en-us/azure/storage/common/storage-service-encryption
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)